snapcraft/plugins/catkin.py

```diff
@@ -66,75 +69,88 @@ def schema(cls):
             },
             'default': [],
         }
+        schema['properties']['source-space'] = {
+            'type': 'string',
+            'default': 'src',
+        }
 
         schema['required'].append('catkin-packages')
 
         return schema
 
     def __init__(self, name, options):
-        super().__init__(name, options)
+        """Sanity-check the plugin config regarding source code paths.
```

Comment:
[`__init__`](http://www.rafekettler.com/magicmethods.html#construction) is much more than this. I am not entirely sure this is the best one liner docstring